content/docs/start/data-and-model-versioning.md

```diff
@@ -160,7 +158,7 @@ run it after `git clone` and `git pull`.
 
 <details>
 
-### ⚙️ Expand to explode the project 💣
+### ⚙️ Expand to delete locally cached data.
```

Comment:
> From https://github.com/iterative/dvc.org/pull/2359#discussion_r612877928:

I don't think we should use the term "cache" here as it's not visible anywhere else, probably intentionally. Maybe keep the original intention for now ("blow up") or go for something like "clear the storage".

Reply:
Er... the word "cache" is used a lot on the same page.